mlir/utils/jenkins/Jenkinsfile

```diff
@@ -678,15 +678,15 @@ pipeline {
                         // Clean up build settings to disable static library and allow
                         // mlir-rocm-runner
                         sh 'rm build/CMakeCache.txt'
-                        buildProject('miopen-gen mlir-miopen-driver mlir-rocm-runner ci-performance-scripts', '')
+                        buildProject('rock-gen mlir-rock-driver mlir-rocm-runner ci-performance-scripts', '')
                     }
                 }
-                stage("Test MLIR vs MIOpen") {
+                stage("Test MLIR vs Rock") {
                     steps {
                         dir('build') {
                             sh 'date --utc +%Y-%m-%d > perf-run-date'
                             // Run MLIR perf benchmarks
-                            sh 'python3 ./bin/MIOpenDriver.py'
+                            sh 'python3 ./bin/RockDriver.py'
```

Comment:
So basically all `MIOpen` names should stay in this file

Reply:
Thanks @zhanglx13 . Does this MIOpenDriver.py refer to the MIOpen project then?

Reply:
MIOpenDriver.py is used to compare the performance between MLIR and MIOpen (tuned vs. untuned). I personally think they should stay as it is. But we do want @jerryyin to take a look here.